plugins/CMakeLists.txt

```diff
@@ -23,7 +26,7 @@ if((((NOT ANDROID AND NOT IOS) AND (UNIX) AND (${LIBOLA_FOUND}) AND (${LIBOLASER
 endif()
 if(UNIX AND NOT ANDROID AND NOT IOS AND NOT MACOS)
     add_subdirectory(dmx4linux)
+    add_subdirectory(spi)
```

Comment:
why have you re-enabled it?
SPI on desktop doesn't make any sense.
Please leave as upstream version

Reply:
OK, I enabled it because spi is enabled in QT .pro file.